src/components/BookingsAndServicesHistoryChart.tsx

```diff
@@ -1,108 +1,90 @@
+import { useBookingsQuery, useServicesQuery } from '@graphql/history/history';
```

Comment:
Minor: Probably a refactor in another pr, you probably want to make a graphql request per page, you can put multiple queries in a single request. Then you would pass down the data to the components so they can be reused on other pages